src/content/docs/concepts/schema.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Schema
+---
+
+A Membrane program's graph is defined by its _schema_, autogenerated in `your-program/memconfig.json` based on the schema you configure in the Explorer panel. The schema is a set of types, which can be _fields_, _actions_, or _events_.
+
+- **Fields**: queryable nodes and values
+- **Actions**: invocable functions on a node
+- **Events**: subscribable notifications
+
+<video src="/cloud-assets/schema.mp4" muted autoplay loop></video>
+
```

Comment:
Do you think it could be helpful to draw parallels with OOP?

Something along the lines of: "Membrane nodes can be thought of objects in the object-oriented programming sense, but unlike normal JavaScript objects, Membrane nodes can be referenced from other programs."

Reply:
> but unlike normal JavaScript objects, Membrane nodes can be referenced from other programs

This part loses me because you _can_ access JS objects from other programs if you import them, say as an npm package, which is similar to membrane program connections/dependencies